xml/net_wicked.xml

```diff
@@ -700,46 +642,83 @@ ip link set down dev eth0
+    </para>
+<screen>&prompt.sudo;ip link set <replaceable>DEV_NAME</replaceable> carrier off</screen>
+    <para>
+     To connect the device back to the physical network, run
+    </para>
+<screen>&prompt.sudo;ip link set <replaceable>DEV_NAME</replaceable> carrier on</screen>
+   </tip>
+<screen>&prompt.sudo;ip link set eth0 up</screen>
    <para>
     After activating a device, you can configure it. To set the IP address, use
-    <command>ip addr
-    add</command>&nbsp;<option><replaceable>IP_ADDRESS</replaceable> + dev
-    <replaceable>DEVICE_NAME</replaceable></option>. For example, to set the
-    address of the interface eth0 to 192.168.12.154/30 with standard broadcast
-    (option <option>brd</option>), enter <command>ip
-    addr</command>&nbsp;<option>add 192.168.12.154/30 brd + dev eth0</option>.
    </para>
+<screen>&prompt.sudo;ip addr add <replaceable>IP_ADDRESS</replaceable> + dev <replaceable>DEV_NAME</replaceable></screen>
+   <para>
+    For example, to set the address of the interface eth0 to 192.168.12.154/30
+    with standard broadcast (option <option>brd</option>), enter
+   </para>
+<screen>&prompt.sudo;ip addr add 192.168.12.154/30 brd + dev eth0</screen>
    <para>
     To have a working connection, you must also configure the default gateway.
-    To set a gateway for your system, enter <command>ip route
-    add</command>&nbsp;<option>gateway_ip_address</option>. To translate one IP
-    address to another, use <command>nat</command>: <command>ip route add
-    nat</command>&nbsp;<option>ip_address</option>&nbsp;<command>via</command>&nbsp;<option>other_ip_address</option>.
+    To set a gateway for your system, enter
+   </para>
+<screen>&prompt.sudo;ip route add gateway_ip_address</screen>
+   <para>
+    To translate one IP address to another, use <emphasis>network address
+     translation (NAT):</emphasis>
+    </para>
+<screen>&prompt.sudo;ip route add nat <replaceable>IP_ADDRESS</replaceable> via <replaceable>OTHER_IP_ADDRESS</replaceable></screen>
+   <para>
+    To display all devices, use
+   </para>
+   <screen>&prompt.sudo;ip link ls</screen>
+   <para>
+    To display the running interfaces only, use
    </para>
+<screen>&prompt.sudo;ip link ls up</screen>
    <para>
-    To display all devices, use <command>ip link ls</command>. To display the
-    running interfaces only, use <command>ip link ls up</command>. To print
-    interface statistics for a device, enter <command>ip -s link
-    ls</command>&nbsp;<option>device_name</option>. To view addresses of your
-    devices, enter <command>ip addr</command>. In the output of the <command>ip
-    addr</command>, also find information about MAC addresses of your devices.
-    To show all routes, use <command>ip route show</command>.
+    To print interface statistics for a device, enter
    </para>
+<screen>&prompt.sudo;ip -s link ls <replaceable>DEV_NAME</replaceable></screen>
+    <para>
+					To view addresses of your devices, enter
+    </para>
```

Comment:
I would suggest to also mention `ip -d link ls`, with recent kernel and iproute2 it shows valuable additional information, especially for virtual devices (vlan, bond, ...).